components/Footer/Footer.js

```diff
@@ -3,74 +3,116 @@ import Link from 'next/link';
 import classNames from 'classnames';
 import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
 import SocialMedia from 'components/SocialMedia/SocialMedia';
+import Button from 'components/_common_/Button/Button';
 import styles from './Footer.css';
 
 function Footer() {
   return (
     <footer className={styles.Footer}>
-      <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
-        <a className={styles.email} href="mailto:contact@operationcode.org">
-          contact@operationcode.org
-        </a>
-
-        <SocialMedia />
-      </div>
-
-      <div className={classNames(styles.footerGrouping, styles.logoGrouping)}>
-        <img src="/static/images/icons/operation_code_logo.svg" alt="Operation Code Logo" />
+      <div className={classNames(styles.footerWrapper, styles.row)}>
+        <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
+          <div className={classNames(styles.logoGrouping)}>
+            <img src="/static/images/operation_code_logo.png" alt="Operation Code Logo" />
+          </div>
+          <div className={classNames(styles.capitalize, styles.marginBottom)}>Connect With Us!</div>
+          <SocialMedia />
+        </div>
 
-        <p className={styles.copyright}>
-          Copyright {`${new Date().getUTCFullYear()}`}
-          <br />
-          Operation Code™
-        </p>
-      </div>
+        <div className={classNames(styles.footerGrouping, styles.linksGrouping, styles.capitalize)}>
+          <div className={styles.linksRow}>
+            <div className={styles.linksColumn}>
+              <Link href="/about">
+                <a>About Us</a>
+              </Link>
+              <Link href="/team">
+                <a>Team</a>
+              </Link>
+              <Link href="/contact">
+                <a>Contact Us</a>
+              </Link>
+              <Link href="/faq">
+                <a>FAQ</a>
+              </Link>
+            </div>
+            <div className={styles.linksColumn}>
+              <Link href="/who_we_serve">
+                <a>Who We Serve</a>
+              </Link>
+              <Link href="/code_schools">
```

Comment:
typo in the route here

Reply:
Sorry I must be missing something... Which one is it?

Reply:
oh jeeze - guess im going blind 😂

Reply:
Haha I thought I was going blind. I think there is only one item left, which is adding more top and bottom for the footer? Do you mean this should be added only when it's on mobile size?

Reply:
Good question! I swore I responded to this. Let's do more vertical margin for the footer period, regardless of viewport.

Reply:
I added top & bottom padding to the footer, hopefully that is enough space.